MdePkg/Include/Protocol/BootManagerPolicy.h

```diff
@@ -31,6 +32,11 @@
     0x113B2126, 0xFC8A, 0x11E3, { 0xBD, 0x6C, 0xB8, 0xE8, 0x56, 0x2C, 0xBA, 0xFA } \
   }
 
+#define EFI_BOOT_MANAGER_POLICY_STORAGE_GUID \
+  { \
+    0XCD68FE79, 0XD3CB, 0X436E, {0XA8, 0X50, 0XF4, 0X43, 0XC8, 0X8C, 0XFB, 0X49 }\

```

Comment:
Has it been defined in Uefi Spec?

Reply:
Yes, it was updated. Its in 2.11,  mantis item: 0002443

https://github.com/UEFI/UEFI_spec/commit/20971078eb7cb0ced138f56f9c7f27d0692d0a61

Reply:
This can be merged after UEFI 2.11 is published.